### PR TITLE
LADX: Update tests to use bases.py

### PR DESCRIPTION
**Pull request by nicholassaylor**
## What is this fixing or adding?
Updates import to bases.py rather than TestBase.py since the latter is deprecated

## How was this tested?
Running unit tests

### CODE REVIEW

**Comment by nicholassaylor** [comment]
Failure is random from Overcooked 2

**Review by NewSoupVi — APPROVED** [review]

**Comment by NewSoupVi** [comment]
LADX still has no official active maintainer so I will merge this without anyone's specific approval being required

**Pull request merged**
